tests/MSBuildDeviceIntegration/Tests/XASdkDeployTests.cs

```diff
@@ -55,7 +55,11 @@ public void DotNetDebug ()
 			AssertHasDevices ();
 
 			XASdkProject proj;
-			proj = new XASdkProject ();
+			proj = new XASdkProject {
+				//TODO: targetSdkVersion="30" causes a crash on startup in .NET 5
+				MinSdkVersion = null,
+				TargetSdkVersion = null,
+			};
```

Comment:
This was needed as discussed here: https://xamarinhq.slack.com/archives/C03CEGRUW/p1596468488148200